support/eula-resources-template.xml

```diff
@@ -27,8 +27,8 @@
 			AAYNRW5nbGlzaCB0ZXN0MQVBZ3JlZQhEaXNhZ3JlZQVQcmludAdT
 			YXZlLi4ueklmIHlvdSBhZ3JlZSB3aXRoIHRoZSB0ZXJtcyBvZiB0
 			aGlzIGxpY2Vuc2UsIGNsaWNrICJBZ3JlZSIgdG8gYWNjZXNzIHRo
-			ZSBzb2Z0d2FyZS4gIElmIHlvdSBkbyBub3QgYWdyZWUsIHByZXNz
-			ICJEaXNhZ3JlZS4i
+			ZSBzb2Z0d2FyZS4gSWYgeW91IGRvIG5vdCBhZ3JlZSwgY2xpY2sg
+			IkRpc2FncmVlLiI=
```

Comment:
Hi!

> $ base64 --decode <<<'AAYNRW5nbGlzaCB0ZXN0MQVBZ3JlZQhEaXNhZ3JlZQVQcmludAdTYXZlLi4ueklmIHlvdSBhZ3JlZSB3aXRoIHRoZSB0ZXJtcyBvZiB0aGlzIGxpY2Vuc2UsIGNsaWNrICJBZ3JlZSIgdG8gYWNjZXNzIHRoZSBzb2Z0d2FyZS4gSWYgeW91IGRvIG5vdCBhZ3JlZSwgY2xpY2sgIkRpc2FncmVlLiI='
> 
> English test1AgreDisagreePrintSave...zIf you agree with the terms of this license, click "Agree" to access the software. If you do not agree, click "Disagree."

Maybe you forgot the trailing full stop (period, `.`)?

Reply:
@pzhlkj6612, you're right, good catch. I have moved the dot as well :)